acrclient/client.py

```diff
@@ -155,3 +156,14 @@ def get_bm_cs_projects_results(
             params=params,
             **kwargs,
         ).get("data")
+
+    def get_external_metadata_tracks(
+        self, params: Optional[GetExternalMetadataTracksParams] = None
+    ) -> list:
+        try:
+            return self.json(path="/api/external-metadata/tracks", params=params).get(
+                "data"
+            )
```

Comment:
it should probably also pass kwargs along so users can pass additional args to `requests.get` (like a session for retry/backoff).
```suggestion
            return self.json(
                path="/api/external-metadata/tracks",
                params=params,
                **kwargs,
            ).get("data")
```

Reply:
Ok good idea